readme.md

```diff
@@ -1,4 +1,6 @@
-# <div style="display: flex;align-items: center;justify-content: center;font-family: sans-serif;text-transform: uppercase;">![Polylang](.github/assets/polylang-logo.png) <span style="font-size: 5.6rem;padding: 0 2rem;">Polylang<span></div>
+<link href=".github/assets/readme.css" rel="stylesheet"></link>
+
+# <div>![Polylang](.github/assets/polylang-logo.png) <span>Polylang<span></div>
```

Comment:
Shouldn't the image have an `aria-hidden="true"` then? Otherwise this will repeat 'Polylang' twice when read through assistive technologies. https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-hidden_attribute